protocol/src/types/primitive.rs

```diff
@@ -174,6 +175,8 @@ impl<'de> Deserialize<'de> for Hex {
     }
 }
 
+pub type Key256Bits = Zeroizing<[u8; 32]>;
```

Comment:
```suggestion
/// Zeroizing is the process of securely zeroing memory to prevent sensitive
/// data from being accessed by unauthorized parties.
///
/// This ensures that sensitive data is securely erased from memory when it is
/// no longer needed.
pub type Key256Bits = Zeroizing<[u8; 32]>;
```